demisto_sdk/commands/content_graph/objects/pack.py

```diff
@@ -303,3 +342,187 @@ def to_nodes(self) -> Nodes:
             self.to_dict(),
             *[content_item.to_dict() for content_item in self.content_items],
         )
+
+    def enhance_pack_properties(self, marketplace: MarketplaceVersions):
```

Comment:
I think we should make a validator for it to enhance those properties automatically.